.bumpversion.cfg

```diff
@@ -0,0 +1,20 @@
+[bumpversion]
+current_version = 2.18.1
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+        (\.(?P<release>[a-z]+)\.(?P<build>\d+))?
+
+serialize = 
+	{major}.{minor}.{patch}.{release}.{build}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+values = 
+	beta
+
+[bumpversion:part:build]
+
+[bumpversion:file:lib/recurly/version.rb]
```

Comment:
For this to work, we need to refactor this file to be a normal string: https://github.com/recurly/recurly-client-ruby/blob/master/lib/recurly/version.rb#L3